2.overview_and_terminology.md

```diff
@@ -306,3 +306,6 @@ This form is not accepted as an alternative for the fully qualified version. It
 | Previous Part        | Next Part          |
 | ------------- |-------------|
 |[1. Purpose And Goals](1.purpose_and_goals.md)| [3. Workload Definition](3.workload.md)|
+
+
+[Relations between various OAM resources](/design/20200727-spec-v0.2.0-data-model.md)
```

Comment:
Please put it at the end of this section: https://github.com/oam-dev/spec/blob/d9e003673696db682974d9e1b3fb9169336460bc/2.overview_and_terminology.md#overview-of-the-model, i.e. `Overview of the Model` section

Reply:
@resouer At the end of `Overview of the Model`.
![alisoc](https://user-images.githubusercontent.com/47811606/88693637-27c99300-d11d-11ea-9b1e-bfc8ad9a8442.png)

Reply:
at green arrow?

Reply:
Yes! Sorry that I missed the notification of this PR.